packages/app/src/app/overmind/namespaces/dashboard/utils.ts

```diff
@@ -1,6 +1,10 @@
 import { SidebarCollectionDashboardFragment as Collection } from 'app/graphql/types';
 import { DELETE_ME_COLLECTION } from './state';
 
+export const isRepoPage = location.pathname.includes('/repositories');
+
+export const repoName = location.pathname.split('/repositories/')[1];
```

Comment:
Same for this, I would pass the `repoName` in to an action instead of gathering it from the URL.

Reply:
I honestly think its better to get it from the URL, in the cases where we are in the repo page this will never break and we already send the page

I think this makes it easier to not screw up

Reply:
this does work when you move pages, I tested this a lot to make sure it wouldn't break but sure I can pass the repo name into the action

Reply:
done